python-api-examples/non_streaming_server.py

```diff
@@ -641,7 +641,11 @@ async def process_request(
         path: str,
         request_headers: websockets.Headers,
     ) -> Optional[Tuple[http.HTTPStatus, websockets.Headers, bytes]]:
-        if "sec-websocket-key" not in request_headers:
+        if "sec-websocket-key" not in (
```

Comment:
Could you also fix `streaming_server.py`?

Reply:
It has been also fixed now.